Arrangement_on_surface_2/include/CGAL/Curved_kernel_via_analysis_2/gfx/Curve_renderer_2.h

```diff
@@ -2258,6 +2258,11 @@ bool compute_double_approx(int var, const NT& l_, const NT& r_,
     pix.yv = CGAL::to_double(engine.y_min + y*engine.pixel_h);
     return ret;
 }
+/*! \copydoc test_neighborhood
+ *  \deprecated please use #test_neighborhood */
+CGAL_DEPRECATED bool test_neighbourhood(Pixel_2& pix, int dir, int& new_dir)
+{ return test_neighborhood(pix, new_dir)' }
```

Comment:
```suggestion
{ return test_neighborhood(pix, new_dir); }
```

Reply:
Oops (for both), thanks for testing and fixing. 
(The problem is that I don't have all packages installed to do a compilation.)